redirects-transition.conf

```diff
@@ -942,6 +1074,14 @@ rewrite ^/agenda/agendas2000/(.*).pdf https://www.fec.gov/resources/updates/agen
 # audio/ broader redirects
 rewrite ^/audio/([0-9]+)/(.*) https://www.fec.gov/resources/audio/$1/$2 redirect;
 
+# audits/ broader redirects for PDF files
+rewrite ^/audits/Hillary_Clinton_for_President/(.*) https://www.fec.gov/resources/legal-resources/enforcement/audits/Hillary_Clinton_for_President/$1;
```

Comment:
It looks like all of the broader redirects for audit PDF files are missing the "redirect;" part at the end.

Reply:
Nice find @dorothyyeager! This has been fixed